scripts/classes/extensions-installer.js

```diff
@@ -75,12 +71,9 @@ class ExtensionsInstaller {
       ...this.localExtensions,
       ...this.extensionsToInstall
     ];
-    _.forEach(excludePackages, packageName => delete packageJsonTemplate.dependencies[packageName]);
     return writePackageJson(packageJsonTemplate)
+      .then(() => linkLocal(process.cwd()))
```

Comment:
maybe we should reverse order of these two? So, I would first call `npm install` and then `linkLocal`, to be sure npm would install all extension dependencies.